src/main/kotlin/com/example/flashcardbackend/flashcard/FlashcardService.kt

```diff
@@ -11,8 +11,15 @@ class FlashcardService(val repository: FlashcardRepository) {
         repository.findAll().map { it.toFlashcardListItemDTO() }
 
     @Transactional(readOnly = true)
-    fun findCardById(id: Int): FlashcardDTO? =
-        repository.findById(id)?.toFlashcardDTO()
+    fun findFlashcardById(id: Int): FlashcardDTO? {
+        val card = repository.findById(id)?.toFlashcardDTO()
```

Comment:
Do you plan to return the HTTP status code 404 if the requested entity isn't found? If so, you need to handle that scenario in this method (and you need to do something else as well). We can talk about this today in our meeting.

Reply:
Yes I missed that very basic thing, because I didn't write tests.